code/model/VirtualPage.php

```diff
@@ -467,8 +454,8 @@ public function hasMethod($method) {
 	 * @return string
 	 */
 	public function castingHelper($field) {
-		if($this->copyContentFrom()) {
-			return $this->copyContentFrom()->castingHelper($field);
```

Comment:
if this was necessary to fix a bug it's a bad sign - methods should be case-insensitive. Recommend reverting this change and fixing upstream if needed.